azurerm/internal/services/powerbidedicated/registration.go

```diff
@@ -0,0 +1,23 @@
+package powerbidedicated
```

Comment:
and this
```suggestion
package powerbi
```
given dedicated doesn't appear anywhere when i google "Power BI Dedicated"

Reply:
updated